app/assets/stylesheets/provider/_pop_navigation.scss

```diff
@@ -35,7 +37,8 @@
     border: $border-width solid $border-color;
     border-radius: $border-radius;
     display: none;
-    margin: line-height-times(1 / 2, true) 0 0 0;
+    margin-top: line-height-times(3 / 2, true);
+    margin-left: calc(-#{$link-padding} + -#{$border-width});
```

Comment:
```suggestion
    margin-left: -calc(#{$link-padding} + #{$border-width});
```

Reply:
@jmprusi doesn’t that work too and make it a bit more readable?

Reply:
It won't work with a preceding -, can't find it in the docs why but the negative has to come from within.